email_template_header/company.py

```diff
@@ -25,8 +25,9 @@ class ResCompany(models.Model):
     _inherit = 'res.company'
 
     def _get_default_header_id(self):
-        header_obj = self.env['email.header']
-        return header_obj._get_header()
+        headers = self.env['email.header'].search([])
+        # return an empty record because of issue https://github.com/odoo/odoo/issues/4384
+        return headers and headers[0] or self.env['email.header']
```

Comment:
Just for the beauty of code, keep the first assign of email.header and use it after to avoid multiple call to self.env:
```python
header_obj = self.env['email.header']
headers = header_obj.search([])
return headers and headers[0] or header_obj
```
What do you think?